mod/layout/set/abstract/bs_badge.rb

```diff
@@ -0,0 +1,9 @@
+format :html do
+  def labeled_badge count, label, opts={}
```

Comment:
bs_badge is gone but the file has still that name

Reply:
Yeah, I assumed there might be multiple badge views.